client/src/components/history/FilterBar.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
```

Comment:
The FilterBar component appears at the top of History. It contains a SearchBar + 2 dropdown menus to filter by expense and date